include/libnuraft/peer.hxx

```diff
@@ -329,6 +346,7 @@ private:
                            ptr<req_msg>& req,
                            ptr<rpc_result>& pending_result,
                            bool streaming,
+                           size_t total_size,
```

Comment:
The name is ambiguous (total size of what?). I'd rather renaming it `req_size_bytes`.